blog.md

```diff
@@ -0,0 +1,73 @@
+# How To Choose The Right Programming Language
+
+If you are new to programming and are wondering which language to learn first, or if you want to get started on a specific project and aren’t sure which language to code it in, here is a brief overview of the most popular programming languages to help get you started.
+
+**HTML and CSS**:
+
+These languages are necessary for creating static web pages. HTML (Hypertext Markup Language) is used to structure all the text, links, and other content you see on a website. CSS is the language used to describe the presentation of Web pages, including colors, layout, and fonts. One of CSS’ key benefits is the way it allows the separation of document content (written in HTML or a similar markup language) from document presentation (written in CSS). Thus, HTML and CSS are markup languages, heavily utilized for creating web pages and web applications.
+
+**JavaScript**:
+
+Along with HTML and CSS, JavaScript forms the basis of front-end web development, facilitating the creation of an interactive element. JavaScript is an object oriented text-based programming language used both on the client-side and server-side that allows you to make web pages interactive.Things like animations, video players, interactive maps, and simple in-browser games can be created with JavaScript. To put it simply, where HTML and CSS are languages that give structure and style to web pages, JavaScript gives web pages interactive elements that engage the user.
```

Comment:
If this is aimed at potentially new programmers, trying to define object oriented programming. Example definition stolen online: Procedural programming (PP), also known as inline programming takes a top-down approach. It is about writing a list of instructions to tell the computer what to do step by step. It relies on procedures or routines. Object-oriented programming (OOP) is about encapsulating data and behavior into objects.

This could be added in the intro paragraph of the blog